docs/clients.md

```diff
@@ -69,6 +69,43 @@ Therefore these two rules may help when choosing what to use:
 * When opening a client within a service, use local clients.  Usually you'll want to open one client per worker.
 * When opening a client from outside a service, such as from inside an actor or some other general use case, use a future client.
 
+## Failure Management
+ 
+### Client Behavior 
+ 
+A Client handles makes a request to an external system.  If any failure occurs
+(example: connection closed, external system isn't responding) the response is 
+sent back as a failure and NOT retried.  The default behavior is to continue
+trying subsequent requests.  If failfast is enabled then all subsequent 
+requests that are queued up in the ServiceClient are removed.  
+
+### Retrying Requests
+
+To enable retries, the LoadBalancingClient (LBC) should be used.  The LBC takes
+in a generator that builds a ServiceClient from an InetSocketAddress and a list
+of InetSocketAddresses.  The LBC defaults to using the Integer max retry as the
+default connection attempts.
+
+Any failure occurring before the the client receives the response (example: 
+connection closed, host not responding) is treated as an attempt and is 
+retried.  If the maximum number of tries are exhausted, a SendFailedException
+is returned with the last exception.
+
+### Example
+
+
+Below is an example of retrying three times to the same host
+
+{% highlight scala %}
+val generator = (address: InetSocketAddress) => {
+  Redis.futureClient(address.toString, 6379)
+}
+val clients = List.fill(3)("myredisserver")
+val lbc = new LoadBalancingClient[Redis](worker, generator, 3, clients)
+
+lbc.send(Command.c(CMD_ZADD, ByteString("key"), ByteString("value")))
```

Comment:
You can actually wrap a LBC inside a redis client, so by doing `val client = Redis.client(lbc)` you can have an LBC but still have the redis client interface and not have to construct commands like this.